homeassistant/components/zwave.py

```diff
@@ -92,6 +93,10 @@
      [COMMAND_CLASS_DOOR_LOCK],
      TYPE_BOOL,
      GENRE_USER),
+    ('rollershutter',
+     [COMMAND_CLASS_SWITCH_MULTILEVEL],
```

Comment:
I started to implement rollershutter support myself a few days ago - but it you were faster ;-) My approach was to add the "specific device class" as another property to identify devices. For the Fibaro rollershutter it has to be MOTOR_CONTROL_CLASS_B. This way dimmers will always be detected as lights, but on the other hand things like garage door openers might also be detected as rollershutters. So its still not perfect, but to detect every motor controller as a rollershutter is still much better than to detect it as a light IMHO. Just take a look at my forks zwave-rollershutter branch...